src/TestHelper/RequestObjectTestHelper.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
Is there any way we could move this into the tests namespace?

Reply:
No, the intention of this test helper is to be used in other project's test cases, and the tests directory of this library is not available for autoloading in a project that depends on it

Reply:
👍